LeaveManagementModule/Backend/app/controllers/employee.js

```diff
@@ -19,15 +42,17 @@ var employee = {
         { res.status(400).json({
              message : "Employee Id Required",
          }); 
-         } 
+         } //Procedure for creating a request 
          sql.connect(mssqlConfig).then(() => {
-             return sql.query('select * from EmployeeLeaves where id = 1');
-         }).then(result => {
+             return sql.query("exec pConditionForLeave @EmployeeId= "+EmployeeId+", @LeaveId="+LeaveId+", @StartDate="+StartDate+", @EndDate="+EndDate+"; ")
```

Comment:
What is EmployeeId , LeaveId , StartDate here ?
You have to get data from request object